### PR TITLE
Update a number of javascript files in common into typescript

### PR DESCRIPTION
**Pull request by ishouty**

### CODE REVIEW

File: common/views/components/FindUs/FindUs.tsx [hunk 1]
```diff
@@ -35,7 +34,7 @@ const StyledFindUs = styled.div.attrs(props => ({
   }
 `;
 
-const FindUs = () => (
+const FindUs: FunctionComponent = (): ReactElement => (
```
Comment (jamesgorrie) [inline common/views/components/FindUs/FindUs.tsx:37]: We don't need the `ReactElement` return type... clutters the file in my 👀

File: common/views/components/ZoomedImage/ZoomedImage.tsx [hunk 2]
```diff
@@ -46,14 +51,19 @@ const ErrorMessage = () => (
...
-type Props = {|
-  id: string,
-  infoUrl: string,
-  setShowViewer: boolean => void,
-  isFullscreen: boolean,
-|};
+type Props = {
+  id: string;
+  infoUrl: string;
+  setShowViewer: (arg: boolean) => void;
```
Comment (jamesgorrie) [inline common/views/components/ZoomedImage/ZoomedImage.tsx:57]: We could name this better.

File: common/views/components/ZoomedImage/ZoomedImage.tsx [hunk 3]
```diff
@@ -85,6 +95,7 @@ const ZoomedImage = ({ id, infoUrl, setShowViewer, isFullscreen }: Props) => {
             },
           ],
         });
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (jamesgorrie) [inline common/views/components/ZoomedImage/ZoomedImage.tsx:98]: Could we just remove the `_`?

**Review by jamesgorrie — APPROVED** [review]
Great! Getting there.

File: common/views/components/FindUs/FindUs.tsx (hunk 1 above)
Comment (jamesgorrie) [reply common/views/components/FindUs/FindUs.tsx:37]: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v16/index.d.ts#L547

You can see the return type is here ☝️

Comment (ishouty) [reply common/views/components/FindUs/FindUs.tsx:37]: nice! ill do that

File: common/views/components/ZoomedImage/ZoomedImage.tsx (hunk 3 above)
Comment (ishouty) [reply common/views/components/ZoomedImage/ZoomedImage.tsx:98]: actually removing this should be fine . !

File: common/views/components/ZoomedImage/ZoomedImage.tsx (hunk 2 above)
Comment (ishouty) [reply common/views/components/ZoomedImage/ZoomedImage.tsx]: renamed this to setShowZoomed

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**